src/judoscale/core/config.py

```diff
@@ -25,12 +23,17 @@ def for_report(self):
         }
 
     def _prepare_logging(self):
+        logger = logging.getLogger('judoscale')
         log_level = logging.getLevelName(self.log_level.upper())
-        logging.basicConfig(
-            level=log_level,
-            format="%(levelname)s - [Judoscale] %(message)s",
-            force=True,
-        )
+        logger.setLevel(log_level)
+
+        if not logger.handlers:
+            stdout_handler = logging.StreamHandler()
+            fmt = "%(levelname)s - [Judoscale] %(message)s"
+            stdout_handler.setFormatter(logging.Formatter(fmt))
+            logger.addHandler(stdout_handler)
```

Comment:
@adamlogic this check of the existing handler fixed the duplicate logs. Because the _prepare_logging method is called once the object is initialized at the bottom, and then from merge method, it was re-adding a handler and when we do that the logs are printed again.

> Note If you attach a handler to a logger and one or more of its ancestors, it may emit the same record multiple times. 
https://docs.python.org/2/library/logging.html#logging.Logger.propagate

Reply:
Nice!